plugins/modules/grafana_user.py

```diff
@@ -270,28 +270,29 @@ def main():
     grafana_iface = GrafanaUserInterface(module)
 
     # search user by login
-    target_user = grafana_iface.get_user_from_login(login)
+    actual_grafana_user = grafana_iface.get_user_from_login(login)
     if state == 'present':
+        has_changed = False
 
-        if target_user is None:
+        if actual_grafana_user is None:
             # create new user
-            user = grafana_iface.create_user(name, email, login, password)
-            module.exit_json(changed=True, user=user)
+            actual_grafana_user = grafana_iface.create_user(name, email, login, password)
```

Comment:
maybe add a check here to test if the new created user is admin ?

Reply:
it's right after line 285

Reply:
at this particular line, the test will always be negative: user creation doesn't allow setting user's permissions

this is done [here](https://github.com/ansible-collections/community.grafana/pull/154/files#diff-3fc8f0dba81b075c6a8ec8b4b3eb63fb97287f9301cfcc599aa0150abd0e8ce9R286)